Demos/Device/LowLevel/CCID/CCID.txt

```diff
@@ -0,0 +1,54 @@
+/** \file
+ *
+ *  This file contains special DoxyGen information for the generation of the main page and other special
+ *  documentation pages. It is not a project source file.
+ */
+
+/** \mainpage CCID Demo
+ *
+ *  \section Sec_Compat Demo Compatibility:
+ *
+ *  The following list indicates what microcontrollers were testes with this demo.
+ *
+ *  \li ATMEGA16U2
```

Comment:
Should expand this to other supported devices (looks like everything except the 8U2 if it requires a lot of USB DPRAM or regular RAM. Follow the template from the other demo text file, please.